repm/types.go

```diff
@@ -42,16 +42,13 @@ type scanRequest struct {
 	db.ScanOptions
 }
 
+type scanResponse = []db.ScanItem
```

Comment:
Type aliases are disrecommended for everyday use. Why doesn't `type scanResponse []db.ScanItem` work?

Reply:
I didn't know about the distinction.